launcher/parameters/nest_to_tvb_parameters.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<settings>
+    <co_simulation_parameters>
+        <title>NEST to TVB parameters</title>
+        <description>Parameters used by nest_to_tvb.py transformer</description>
+        <variables>
+            <!-- no variables defined -->
+            <var_000>
+                <var_name>__no_variable_name__</var_name>
+                <var_value>__no_variable_value__</var_value>
+            </var_000>
```

Comment:
The same as before: If I understand this correctly, here would be the variables (and below parameters) for the nest_to_tvb.py example.

I will try to complete my list of variables and parameters soon, so we can discuss what information you need from me!

Reply:
1. I must add some remark here, just because the configuration_manager returns a "str" object instead of a dictionary when the <var_000></var_000> section was empty, I added these values just to mimic that something has been included even though they are kind of placebo to the launcher.
2. Variables is the mechanism to change in some how the launcher behavior when the actions are launched, mostly related to how to bring environment or launcher variables into the context of each action.
3. In order to change the number the spikes for instance, the <parameters></parameters> sections must be used.